mypy/checker.py

```diff
@@ -2410,6 +2410,10 @@ def check_multi_assignment_from_tuple(self, lvalues: List[Lvalue], rvalue: Expre
                                                            reinferred_rvalue_type, context,
                                                            infer_lvalue_type)
                     return
+                if isinstance(reinferred_rvalue_type, AnyType) and self.current_node_deferred:
+                    # Doing more inference in deferred nodes can be hard, so give up for now.
+                    # TODO: should we try harder?
```

Comment:
I don't think that we need the TODO, since doing inference in deferred nodes is pretty pointless.